examples/prefect/model_data_benchmark.py

```diff
@@ -0,0 +1,23 @@
+import pickle
```

Comment:
I would say merge the files into one - the tags provide acceptable semantic grouping already.